packages/python/plotly/_plotly_utils/basevalidators.py

```diff
@@ -2645,30 +2640,28 @@ def validate_coerce(self, v, skip_invalid=False, _validate=True):
                     v_el = v_el.to_plotly_json()
 
                 if isinstance(v_el, dict):
-                    v_copy = deepcopy(v_el)
-
-                    if "type" in v_copy:
-                        trace_type = v_copy.pop("type")
-                    elif isinstance(v_el, Histogram2dcontour):
```

Comment:
I believe this check never hit as in in line 2647 is checked whether `v_el` is a dict.